src/opera/commands/diff.py

```diff
@@ -0,0 +1,203 @@
+import argparse
+import typing
+import yaml
+import json
+import tempfile
+
+from pathlib import Path, PurePath
+from os import path
+
+from opera.error import DataError, ParseError
+from opera.parser import tosca
+from opera.storage import Storage
+from opera.compare.template_comparer import TemplateComparer, TemplateContext
+from opera.compare.instance_comparer import InstanceComparer
+from opera.utils import format_outputs, save_outputs
+
+
+def add_parser(subparsers):
+    parser = subparsers.add_parser(
+        "diff",
+        help="Compare TOSCA service template to the one from"
+             "the opera project storage and print out their differences"
+    )
+    parser.add_argument(
+        "--instance-path", "-p",
+        help=".opera storage folder location"
+    )
+    parser.add_argument(
+        "--inputs", "-i", type=argparse.FileType("r"),
+        help="Optional: YAML file with inputs to "
+             "override the inputs supplied in init",
```

Comment:
This can be updated to something like:

```suggestion
        help="Optional: YAML or JSON file with inputs "
             "that will be used along with the comparison",
```